src/canvas-force-graph.js

```diff
@@ -160,10 +161,20 @@ export default Kapsule({
         ctx.restore();
       }
 
+      function parseLineDashSegments(segments) {
+        if (typeof(segments) == 'string') {
```

Comment:
How can `segments` be a `string`? Isn't it supposed to always be an array of numbers? Perhaps we don't need this parsing step, which may save some performance.

Reply:
The `[5, 5]` I pass as the dash gets stringified to `"5,5"` (by indexBy probaby?)

Reply:
With confirmation that `indexBy` stringifies arrays, this function has been removed and the existing `parseLineDashSegments` call replaced with the `join`/`parseFloat` expression if the stringified array given is not `''` or `'null'`